mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -464,19 +465,23 @@ void registrationsShouldNotHandleEventsAfterStop() throws Exception {
                 MailboxListenerCountingSuccessfulExecution listener = new MailboxListenerCountingSuccessfulExecution();
                 eventBus.register(listener, GROUP_A);
 
-                ConcurrentTestRunner.builder()
+                try (Closeable closeable = ConcurrentTestRunner.builder()
                     .operation((threadNumber, step) -> eventBus.dispatch(EVENT, KEY_1))
                     .threadCount(THREAD_COUNT)
                     .operationCount(OPERATION_COUNT)
-                    .runSuccessfullyWithin(Duration.ofMinutes(1));
+                    .run()) {
```

Comment:
no need to check for errors?

Reply:
oh I see you changed the meaning of run. You don't fear to break other run calls?

Reply:
no, I exposed `.run()`, previous methods are still there